package.json

```diff
@@ -75,9 +75,9 @@
     "repo:resolve": "node scripts/set-resolution.js",
     "start-graphiql": "yarn workspace graphiql dev",
     "start-monaco": "yarn workspace example-monaco-graphql-webpack start",
-    "t": "yarn run test",
+    "t": "yarn test",
     "test": "yarn jest",
-    "test:ci": "yarn build:cm6-graphql && jest",
+    "test:ci": "yarn build && jest --coverage && yarn workspace monaco-graphql test",
```

Comment:
now I need to build some packages for `vite` example, it can be improved with pnpm migration `injected: true` https://github.com/shuding/nextra/blob/e4036433d1221f5256456b09eb2f25956629b41f/examples/swr-site/package.json#L28

Reply:
right now my priority is introducing end to end tests for vscode-graphql, and more importantly generating a source of income for myself. I don't anticipate being able to work on the pnpm migration until the fall. Meanwhile, if you can upgrade tsdoc to the latest, and change the bundling to work as such, that may speed up the process

Reply:
sure! I can pick `pnpm` migration and we have pnpm heroes in The Guild such as @enisdenjo, @YassinEldeeb and @tuvalsimha so I could ask for help for them 😜